### PR TITLE
ui: Update placeholders for adding new tier

### PR DESCRIPTION
**Pull request by davidjumani**
### Description

Fixes https://github.com/apache/cloudstack/issues/5293

Adds a more understandable description for adding new network tiers

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

### Screenshots (if appropriate):

Screenshot from 2021-08-17 13-24-53

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan ui

File: ui/public/locales/en.json [hunk 1]
```diff
@@ -640,8 +640,6 @@
 "label.create.account": "Create Account",
 "label.create.backup": "Start Backup",
 "label.create.network": "Create New Network",
-"label.create.network.gateway.description": "The gateway of the tier in the super CIDR range and not overlapping the CIDR of any other tier in this VPC.",
-"label.create.network.netmask.description": "Netmask of the tier. For example, with VPC CIDR of 10.0.0.0/16 and network tier CIDR of 10.1.1.0/24, gateway is 10.1.1.1 and netmask is 255.255.255.0",
```
Comment (utchoang) [inline ui/public/locales/en.json:644]: @davidjumani I found the keys, labels are not translated in Networks -> VpcTiersTab. It doesn't seem right that you delete these keys?
image

Comment (davidjumani) [reply ui/public/locales/en.json:644]: @utchoang Forgot to add the vue file. Can you please check now

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
It's works. Thanks @davidjumani 
image

**Comment by nvazquez** [comment]
@davidjumani can you target this PR for branch 4.15?

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by shwstppr — APPROVED** [review]

**Comment by rhtyd** [comment]
@davidjumani minor nit - we probably don't want the colon : after parameter names, see other forms

**Comment by utchoang** [comment]
@davidjumani 
In darkmode view, please remove `style="color: rgba(0,0,0,.45)"` in info icon.
image

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]

**Review by nvazquez — APPROVED** [review]

**Pull request merged**